server/lib/version.js

```diff
@@ -22,17 +22,11 @@
 const cp = require('child_process');
 const fs = require('fs');
 const path = require('path');
-const Promise = require('bluebird');
```

Comment:
most changes in this file are with the purpose to remove async stuff and call version on `fxa-content-server.js` start. then we can access version anywhere and not worry about adding promises in scripts and webpack config.